proto/regen/ecocredit/v1alpha1/events.proto

```diff
@@ -38,6 +38,14 @@ message EventCreateBatch {
   // end_date is the end of the period during which this credit batch was
   // quantified and verified.
   string end_date = 6 [ (gogoproto.moretags) = "yaml:\"end_date\"" ];
+
+  // project_location is the location of the project backing the credits in this
+  // batch. It is a string of the form
+  // <country-code>[-<sub-national-code>[ <postal-code>]], with the first two
```

Comment:
```suggestion
  // <country-code>[-<sub-national-code>][-<postal-code>], with the first two
```

Reply:
1. I  think we can have postal code without sub-national code.
2. shouldn't we have `-` before postal code as well?

Reply:
I think it's okay. We've already done many design cycles on this and I don't see a reason to change now

Reply:
So, to confirm: based on the change in this PR: sub national code is required if one wants to put postal code. Correct?